packages/react-navigation/src/__tests__/rum/instrumentation/DdRumReactNavigationTracking.test.tsx

```diff
@@ -154,6 +154,35 @@ describe.each([
                 );
             });
 
+            it('sends a related RUM ViewEvent when switching screens { viewPredicate returns null }', async () => {
+                // GIVEN
+                const navigationRef = createRef<any>();
+                const { getByText } = render(
+                    <FakeNavigator1 navigationRef={navigationRef} />
+                );
+                const goToAboutButton = getByText('Go to About');
+                const predicate: ViewNamePredicate = function (
+                    _route: Route<string, any | undefined>,
+                    _trackedName: string
+                ) {
+                    return null;
+                };
+                DdRumReactNavigationTracking.startTrackingViews(
+                    navigationRef.current,
+                    predicate
+                );
+
+                // WHEN
+                expect(goToAboutButton).toBeTruthy();
+                fireEvent(goToAboutButton, 'press');
+
+                // THEN
+                expect(DdRum.startView).not.toHaveBeenCalledWith(
```

Comment:
shouldn't we just check that there were no interactions with `DdRum.startView` at all? Because maybe there is a logic in the class which makes some fallback, and calls `startView` with some other params - in this case test will pass, while it shouldn't.

And I guess we don't also want to have the calls with `Home` here as well , given that predicate rejects everything.